roles/test-utils/mining-device/src/main.rs

```diff
@@ -1,10 +1,14 @@
-use async_std::net::TcpStream;
 use key_utils::Secp256k1PublicKey;
-use network_helpers_sv2::Connection;
+use network_helpers_sv2::noise_connection_tokio::Connection;
 use roles_logic_sv2::utils::Id;
-use std::{net::SocketAddr, sync::Arc, thread::sleep, time::Duration};
+use std::{
+    net::{SocketAddr, ToSocketAddrs},
+    sync::Arc,
+    thread::sleep,
```

Comment:
`thread::sleep` is used in the `connect` method which uses `tokio` as well, so I think it would be better to use `tokio::time::sleep` here

Reply:
yea I didn't see it ty